### DISCUSSION TITLE
Things to consider when selling hosted Homer?

### DISCUSSION THREAD

**Discussion by tipsy** [sipcapture/homer#542] [General]
I'm interested in selling a hosted homer solution to telecom operators. Is there anything in the license which prevents me from doing this, or something I have to be aware of?

**Comment by tite09-ops** [comment]
I think you will need to filter view in oder to be sure that each customer see only calls related to his sip domain. how can you do that with homer 7, do you have an idea ?

**Comment by lmangani** [comment]
> I'm interested in selling a hosted homer solution to telecom operators. Is there anything in the license which prevents me from doing this, or something I have to be aware of?

Yes. Contact us at office@qxip.net and we'll share further details.
